### PR TITLE
intro_concurrency: Fix broken links

### PR DESCRIPTION
**Pull request by mewmew**

### CODE REVIEW

**Comment by kortschak** [comment]
LGTM (leaving for @btracey to merge)

**Comment by btracey** [comment]
Thanks.

File: 2015/intro_concurrency.slide [hunk 1]
```diff
@@ -365,8 +365,8 @@ DEMO (github.com/btracey/mpi/examples/helloworld)
 * More information
 Reference Material
 
-- [[golang.org]]
-- [[github.com/gonum]]
+- [[http://golang.org][golang.org]]
```
Comment (sbinet) [inline 2015/intro_concurrency.slide:368]: why not use the `https` version here as well?

**Comment by mewmew** [comment]
> why not use the https version here as well?

Done as part of #10.

**Pull request merged**